client/my-sites/plans/plan-overview/style.scss

```diff
@@ -0,0 +1,33 @@
+.plan-overview {
+	.plan-feature__button,
+	.plan-status__button {
+		flex-shrink: 0;
+		margin-left: 15px;
+		min-width: 150px;
```

Comment:
I don't think we should add such rule. It's a red herring - it works mostly when the language is set to English. I'll try to see what we can do here but aligning buttons in a flexbox layout is generally quite difficult to achieve. Maybe we can use a table layout in that case.